ui/component/event-inline.component.ts

```diff
@@ -1,21 +1,21 @@
-import { GetEventDto } from "../../server/events/events.dto"
-import { Column, Heading2, Heading4, Link, Paragraph, Row, RowSpaced, Span, Text } from "../core/html.elements"
-import { Component } from "../core/html.interfaces"
+import { EventModel } from "../../server/events/event.model";
+import { Link, RowSpaced, Text } from "../core/html.elements";
+import { Component } from "../core/html.interfaces";
 
-export const EventInline: Component<GetEventDto> = (event: GetEventDto) => {
+export const EventInline: Component<EventModel> = (event: EventModel) => {
 
-    const icon = event.ratingPercentage >= 50 ? "✔️": "❌";
+    const icon = event.ratingPercentage >= 50 ? "✔️" : "❌";
 
     return RowSpaced(
         Text(
             Link({ text: event.name, href: `/events/${event.id}` }),
             Text(` (${event.sportName})`),
         ),
         Text(
-            event.totalRatings > 0 
+            event.totalRatings > 0
```

Comment:
Does this check also belong to "business logic" regarding an `EventModel`?
If so, maybe we should also delegate it to a separate field/method of the `EventModel` model, e.g. `statisticsAvailable` or `ratingsAvailable`